pyproject.toml

```diff
@@ -32,23 +32,26 @@ classifiers = [
     # that you indicate whether you support Python 2, Python 3 or both.
     'Programming Language :: Python :: 3',
     'Programming Language :: Python :: 3.7',
-    'Programming Language :: Python :: 3.8'
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11'
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8.1,<3.10"
-awscli = ">=1.25.36"
-boto3 = "^1.26.133"
-botocore = "^1.27.36"
+python = ">=3.8.1,<3.12"
```

Comment:
3.8.1 or 3.8? See my comments on cgap-portal.

Reply:
I didn't change that from master, it used to be: python = ">=3.8.1,<3.10" ... not sure why it would be 3.8.1 ...
(Same comment from cgap-portal ... and Will replied: I think I recall from a long time ago 3.8.1 has some specific issues with cgap-portal IRC hence the explicit exclusion)